### DISCUSSION TITLE
Upgrade available in `winget list` but not in `winget upgrade X`

### DISCUSSION THREAD

**Discussion by MeikTranel** [microsoft/winget-cli#1391] [Q&A] [answered]
I've encountered a couple of packages that show off as having updates available but fail when attempting to upgrade them:
## Result: `winget list`
![image](https://user-images.githubusercontent.com/6466560/130335416-586faac4-a866-4e5b-b49e-2eb09944098a.png)

## Result: `winget upgrade Powertoys`
![image](https://user-images.githubusercontent.com/6466560/130335445-0fa40cb3-1e6a-4121-a329-cc54bc0d87d8.png)

I'm super unsure whether this is expected behavior since Powertoys does have an odd self-updater situation going on.

**Comment by ImJoakim** [comment]
The `ProductCode` for version `0.43.0` is incorrect, I'll make a PR to add the correct `ProductCode`.

**Reply by ImJoakim** [reply]
https://github.com/microsoft/winget-pkgs/pull/25653

**Reply by ImJoakim** [reply]
Well, the `ProductCode` was incorrect, but it won't fix the issue. See https://github.com/microsoft/winget-cli/issues/1191#issuecomment-870667044

**Comment by denelon** [comment, marked as answer]
We know exactly what is happening with PowerToys. This was reported originally in #752. The fix is a bit more complicated for that one #1242. We're currently working on the v1.1 schema so hopefully this will be fixed soon[tm].

Duplicate of #752
